scripts/test-runner.js

```diff
@@ -143,16 +151,29 @@ const shutdownRC = () => {
     }
 }
 
-if (process.argv.length === 3) {
+if (process.argv.length === 3 || process.argv.length === 4) {
     if (process.argv[2] === 'unit') {
-        testCommand = 'node node_modules/mocha/bin/mocha "test/unit/**/*.js"';
+        if (process.argv.length === 4) {  //
```

Comment:
Thought it would be good to add this functionality. We can run tests by regex using npm scripts like this:

`npm run test PNCounterBasicTest`

or run all tests as default behaviour as before
`npm run test`